textattack/augmentation/augmenter.py

```diff
@@ -11,40 +15,100 @@ class Augmenter:
             that suggests new texts from an input.
         constraints: (list(textattack.Constraint)): constraints
             that each transformation must meet
+        words_to_swap: (int): Number of words to swap per augmented example
```

Comment:
So the augmenter only works with word swaps for now?

Reply:
Also maybe name this num_words_to_swap?

Reply:
fixed and added a comment